guide/payments/privacy.md

```diff
@@ -0,0 +1,139 @@
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/privacy/header.jpg"
+    retina = "/assets/images/guide/payments/privacy/header@2x.jpg"
+    mobile = "/assets/images/guide/payments/privacy/privacy-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/privacy/privacy-header-mobile@2x.jpg"
+    alt-text = "Illustration of an eye, with a cross on top of an ink blot background"
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+
+# Transaction privacy
+
+<!--
+This page should inform about what information is made public when sending or receiving, how the application can help minimize unnecessary privacy leaks, basic application functionality to help, and when we can, more advanced options.
+- Single-use addresses (avoiding address reuse)
+- Coin selection / labelling
+- Coin join / Pay join
+-->
+
+It’s a common misconception that Bitcoin is anonymous. Instead, it should be referred to as pseudonymous which means each address you generate is like a new identity. Unless you reveal that you are the owner whether directly or indirectly you are able to remain anonymous.
+
+Transactions, their signatures, and addresses after they are mined remain public on the ledger forever. However, there is no personally identifiable information attached to transactions on the blockchain. The transparent property of the blockchain allows anyone to easily verify the amount of bitcoin in existence which means it also makes looking up any address or transaction trivial. This can be demonstrated going back to the first block on [January 3, 2009](https://blockstream.info/tx/4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b).
+
+> Each Bitcoin transaction contains at least one input and at least one output. This means that once a single address is known, there is a trail to follow the bitcoin.
+> 
+> <cite>As documented by <a href="https://docs.wasabiwallet.io/FAQ/FAQ-GeneralBitcoinPrivacy.html#how-is-bitcoin-bad-in-terms-of-privacy">Wasabi Wallet</a></cite>
+
+The key to keeping your transactions private is to prevent others from determining which addresses you own[^3]. Satoshi let others know they mined the first block, which contained just one transaction, with the block reward getting sent to their address. This illustrates the possible permanence of association between an address and an identity. That being said, the pseudonym “Satoshi Nakamoto” also contributes to further anonymity. While it's possible break asumptions of ownership going forward, the challenge is to recover privacy once an association is made public.
+
+<!-- TODO: add graphic and heading that demonstrate points of compromise when transacting with bitcoin -->
+
+## Methods to preserve privacy
+
+Keeping private takes constant and diligent work but is not impossible to ensure an acceptable level of privacy[^1]. Let’s explore some methods and practices that help preserve privacy on a transaction level.
```

Comment:
I'd change this to ...Keeping Bitcoin payments private... good to keep the context going - things isn't as clear.

Reply:
do you mean it's not focused and clear since it could mean; bitcoin privacy, bitcoin transaction privacy, or general privacy?

i think the statement stands true to any of the above still.

Reply:
The page context is clear, and 'keeping things private' is better than 'keeping private'.
I'd say commit.